src/test/acceptance/features/cookies.feature

```diff
@@ -8,3 +8,14 @@ Feature: Cookies page
     When I click the Cookies link
     Then the cookies page is shown
     And all page content is present on the cookies page
+
+  Scenario: The back link on the Cookies page is not shown if we navigate to it directly
+    When I go directly to the cookies page
```

Comment:
You should now be using the same language as our new tests, so this should be `I navigate to the cookies page`, there is already a step you can use.

Reply:
Updated.